decoder.go

```diff
@@ -357,6 +363,19 @@ func (d *decoder) getStrLen(tag byte) (int32, error) {
 		return -1, newCodecError("getStrLen")
 	}
 }
+func (d *decoder) readInstance2(cls ClassDef) (interface{}, error) {
+	var tmpMemberMap = make(map[string]interface{})
+
+	for i := 0; i < len(cls.FieldName); i++ {
+		fldName := cls.FieldName[i]
+		obj, err := d.ReadObject()
+		if err != nil {
+			fmt.Println("struct error", err)
```

Comment:
do you need to add obj to map if the error occurred? Maybe it has the sense to add `continue` here